### PR TITLE
Gives Sergeant-At-Armsky Unique Flavor Text/Description

### PR DESCRIPTION
**Pull request by NotZang**
I noticed he didn't have his own little flavor text like all the other variants of Beepsky, so I thought I'd maybe give him one.

## About The Pull Request
This is just a simple addition of a unique description tag on the one roundstart securitrons that seemed to be missing one, and instead used Commander Beep O'Sky's.

## Why It's Good For The Game
Each of the other variants of Beepsky have their own unqiue little description attached to them. Officer Beepsky, Prison Ofitser, etc. Sergeant-At-Armsky is the only one that does not possess a unique tag at this time, instead directly drawing from Commander Beep O'Sky's description. Having his own description consistent with the other flavor blurbs accompanying the other bots completes the so called "family" picture. A small detail.

## Changelog

:cl:
qol: Consistency with the other bot's examination text.
/:cl:

### CODE REVIEW

**Review by JohnFulpWillard — APPROVED** [review]

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -63,6 +63,7 @@
 
 /mob/living/simple_animal/bot/secbot/beepsky/armsky
 	name = "Sergeant-At-Armsky"
+	desc = "It's Sergeant-At-Armsky! A disgruntled assistant to the Warden. He would probably shoot you if he had hands."
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:66]: when you examine something, isn't the first line already "its [thingy]!"

Comment (JohnFulpWillard) [reply code/modules/mob/living/simple_animal/bot/secbot.dm:66]: It's current state
image

Comment (tf-4) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:66]:
```suggestion
	desc = "It's Sergeant-At-Armsky, a disgruntled assistant to the warden. He would probably shoot you if he had hands."
```
```suggestion
	desc = "It's Sergeant-At-Armsky! He's a disgruntled assistant to the warden that would probably shoot you if he had hands."
```
Slightly clunky wording imo, got options here.

Comment (NotZang) [reply code/modules/mob/living/simple_animal/bot/secbot.dm:66]: Yeah, that's right. Before the edit he had no description line, and was drawing from the top most variant of Beepsky, which happens to be Beep O'sky.

Comment (NotZang) [reply code/modules/mob/living/simple_animal/bot/secbot.dm]: I like that second one. Thanks man.

**Pull request merged**